### PR TITLE
task trigger testing

### PR DESCRIPTION
**Pull request by datamel**
Increased code coverage

This is a small change with no associated Issue.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Appropriate tests are included (unit and/or functional).
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

### CODE REVIEW

File: cylc/flow/tests/test_task_trigger.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import pytest
```
Comment (kinow) [inline cylc/flow/tests/test_task_trigger.py]: Missing license header I think

File: cylc/flow/task_trigger.py [hunk 2]
```diff
@@ -106,7 +96,8 @@ def get_trigger_name(trigger_name):
         for standard_name, alt_names in _ALT_TRIGGER_NAMES.items():
             if trigger_name == standard_name or trigger_name in alt_names:
                 return standard_name
-        raise TaskTriggerError("Illegal task trigger name: %s" % trigger_name)
+        raise TriggerExpressionError(
+            "Illegal task trigger name: %s" % trigger_name)
```
Comment (kinow) [inline cylc/flow/task_trigger.py]: If you update the PR, maybe f-string this one?

**Comment by kinow** [comment]
Sensitive tests failed on travis, but unit tests passed with no issues.

**Comment by oliver-sanders** [comment]
Task trigger now has [100% coverage](https://codecov.io/gh/cylc/cylc-flow/compare/da273be5d66ee78ae1271c4f5836f7293a87116e...42386f894d7805d3c0f4ab0ab315c45061022c32/src/cylc/flow/task_trigger.py).

File: cylc/flow/tests/test_task_trigger.py (hunk 1 above)
Comment (datamel) [reply cylc/flow/tests/test_task_trigger.py]: No probs, I'll sort that.

Nice to meet you all too  :)

File: cylc/flow/task_trigger.py (hunk 2 above)
Comment (oliver-sanders) [reply cylc/flow/task_trigger.py]: https://www.python.org/dev/peps/pep-0498/

**Pull request merged**